components/scream/tests/coupled/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 # NOTE: if you have baseline-type tests, add the subdirectory OUTSIDE the following if statement
 if (NOT ${SCREAM_BASELINES_ONLY})
-  if ("${SCREAM_DYNAMICS_DYCORE}" STREQUAL "HOMME")
-    add_subdirectory(homme_physics)
-  endif()
   if (SCREAM_DOUBLE_PRECISION)
-    add_subdirectory(shoc_cld_p3_rrtmgp)
+    add_subdirectory(shoc_cld_spa_p3_rrtmgp)
+    if ("${SCREAM_DYNAMICS_DYCORE}" STREQUAL "HOMME")
+      add_subdirectory(homme_shoc_cld_spa_p3_rrtmgp)
```

Comment:
I guess the name of this folder follows the other one. I'm not super fond of listing all atm procs in the name, since it makes the name long and hard to parse. But I won't raise a wall here.